README.md

```diff
@@ -1,42 +1,11 @@
-### Bootstrapping
+# Internet of Things API
 
-Clone this repo and run `yarn bootstrap`. This will automatically initialize a fresh repository and install the dependencies listed in `package.json`
-
-Run `yarn test`. You should see a single test run and pass in the console.
-
-Request a new repository from your mentor. They should follow the convention `<user>-node-js-apprenticeship`. Add this new repo as your origin, e.g.:
-
-`git remote add origin https://github.com/moove-it/<user>-node-js-apprenticeship.git`
-
-Make your initial commit and push to the new remote repository:
+This is an API that registers a new IOT device.
 
+**To run:**
 ```
-git add .
-git commit -m "Initial commit"
-git push -u origin master
+Start the server:
+$ node server.js
+Make a post:
+$ node Main [newDeviceName]
```

Comment:
let's also write this as a script, and since this client's purpose is simply to exercise the api, let's also name it something more informative, like `client.js`, rather than `Main`.

Reply:
done